app/js/shared/ucBackground/ucBackground.directive.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
Ok, that didn't come to my mind. I thought using `var` was a way to tackle that, but minifying changes that too.